### ISSUE TITLE
Ballista 43.0.0 Release

### ISSUE THREAD

**Issue by andygrove** [apache/datafusion-ballista#974]
**Is your feature request related to a problem or challenge? Please describe what you are trying to do.**

We have a milestone tracking items for the next release: https://github.com/apache/datafusion-ballista/milestone/4

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by andrewwebber** [comment]
It would bring great confidence to the community and or unsure observers if there was a regular release cadence. Would you have an update regarding this ticket?

**Comment by milenkovicm** [comment]
As balista has tight coupling with datafusion, and we cant really mix and match different versions, would it make sense to have same version as datafusion, like datafusion-python does? it would help users to pick right datafusion version (probably we should re-export datafusion in `ballista::datafusion`)

main reason we can't have same version as datafusion is that ballista is still not as mature as df is, so we should not use major version > 0. Nothing stopping us to use keep minor version in sync with df major version. This would users with decision which versions to choose

in short, proposal is: 

- use `0.43.0` for next release and keep minor version in sync with datafusion major
- use same version as datafusion once ballista get promoted to version 1

wdyt @andygrove ?

**Comment by andygrove** [comment]
I don't have a strong opinion, but if we are going to change the versioning scheme, I would rather do it once than twice. Why don't we just jump straight to 43.0.0 for the next release. This would be consistent with the other subprojects.

**Comment by milenkovicm** [comment]
I have no objections. Other projects put the bar very hight, we might get under pressure 😀

**Comment by andygrove** [comment]
Sure, lets do it. Can you create a PR against `main` to update version numbers and add the changelog?
